src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -133,8 +136,13 @@
     </Window.Resources>
 
     <!--This is the main row of the Preference Window-->
-    <Grid x:Name="mainGrid"
-    	  Background="{StaticResource PreferencesWindowBackgroundColor}">
+    <Border x:Name="mainBorder"
+            BorderBrush="{StaticResource WorkspaceBackgroundHomeBrush}"
```

Comment:
I guess you happen to found out this color from other places is white?

Reply:
I got this one from the NodeView border https://github.com/DynamoDS/Dynamo/blob/b02d0349fc162d4f63beb9c5eea92400067496a2/src/DynamoCoreWpf/Views/Core/NodeView.xaml#L569 as instructed by @Amoursol. The only thing is the borderthickness, which for the Nodes is 1px, and for the PreferencesView is now set at 2px. I will double-check with @Amoursol to see if there is preference for each maybe.

Reply:
Thanks, will merge it for now